CONTRIBUTING.md

```diff
@@ -418,6 +418,9 @@ Be sure to post in the PR conversation that you have made the requested changes.
 
 ### Other resources
 
+-   [Challenge Style Guide to create a new freeCodeCamp
```

Comment:
How about "Challenge Style Guide to create new or update existing freeCodeCamp challenges" ??